app/reward-nft/page.tsx

```diff
@@ -0,0 +1,26 @@
+import { EVENTS } from "@/constants/constants"
+import Events from "../components/nft_events_rewards/EventBadges"
+import { REWARD_DATA } from "@/constants/constants"
+import TokenRewards from "../components/nft_events_rewards/TokenRewards"
+import { CERTIFICATIONS } from "@/constants/constants"
+import NFTCertifications from "../components/nft_events_rewards/NftCertifications"
+
+export default function Page() {
+
+
+    return (
+        <>
+
```

Comment:
it is supposed to be one component that takes different data and has a tab so that when i click on certfications, it shows me the certifications, if i click on rewards i see rewards. its not  supposed to be 3 different components. 

Please fix as needed

Reply:
Alright